changelog/unreleased/enhancement-remove-uikit

```diff
@@ -0,0 +1,7 @@
+Enhancement: Remove UiKit
```

Comment:
This is a breaking change for  extension developers. Please mark it as `change` instead of `enhancement`.